bip122/caip2.md

```diff
@@ -57,14 +57,26 @@ compatible with the reference syntax defined above.
 This is a list of manually composed examples:
 
 ```
-# Bitcoin mainnet (see [chain ID section of BIP122](https://github.com/bitcoin/bips/blob/master/bip-0122.mediawiki#definition-of-chain-id))
+# Bitcoin mainnet (see https://github.com/bitcoin/bips/blob/master/bip-0122.mediawiki#definition-of-chain-id)
 bip122:000000000019d6689c085ae165831e93
 
-# Litecoin
+# Bitcoin Cash mainnet (forked from Bitcoin mainnet after block 478560, with "genesis" hash of 000000000000000000b15ad892af8f6aca4462d46d0b6e5884cadc033c8f257b)
+bip122:000000000000000000b15ad892af8f6a
```

Comment:
I think the comment should be "after block 478559", or even "after block 478558". It's not entirely clear what block number should be used.

The [Wikipedia article](https://en.wikipedia.org/wiki/Bitcoin_Cash) says that _split block_ is 478559, and I initially assumed that the first block *after* fork is 478560. However, [Bitcoin Cash site](https://bitcoincash.org/) suggests that last block before the fork was 478558: 

>Anyone who held Bitcoin at that time (block 478558) became an owner of Bitcoin Cash (BCH).

Reply:
Correct, Bitcoin Cash came into existence at block height 478559.

Reply:
While https://www.blockchain.com/explorer/blocks/bch/478558 is marked as the block when it split.

Reply:
Block 478558 was the last common block. Block 478559 was the first Bitcoin Cash block, with hash `000000000000000000651ef99cb9fcbe0dadde1d424bd9f15ff20136191a5eec`. I believe `bip122:000000000000000000651ef99cb9fcbe` should be the BIP122 identifier.